### PR TITLE
cmd/verify: remove cask logic entirely

### PR DESCRIPTION
**Pull request by woodruffw**
This removes the Cask logic entirely, since it's entirely a no-op: we iterate with `named.to_formulae` instead.

### CODE REVIEW

**Comment by woodruffw** [comment]
Confirmed that this works:

```
arbet:homebrew-brew-verify william$ brew verify xz
==> Downloading https://ghcr.io/v2/homebrew/core/xz/blobs/sha256:01ced87d92d0c1131c069108efb14f6940f9e528e2d044ac
Already downloaded: /Users/william/Library/Caches/Homebrew/downloads/a8def111ef35d2c9d46e56ff2c6c7b623b61be0f395014454e9f11db2db16981--xz--5.4.6.arm64_sonoma.bottle.tar.gz
Verifying attestations for the artifact found at file:///Users/william/Library/Caches/Homebrew/downloads/a8def111ef35d2c9d46e56ff2c6c7b623b61be0f395014454e9f11db2db16981--xz--5.4.6.arm64_sonoma.bottle.tar.gz
All attestations have been successfully verified!
```

However, we'll wait to merge this after the backfill verification work is merged.

**Review by josephsweeney — APPROVED** [review]
I accidentally pulled in the changes in main to my branch, so actually I need to merge this in to fix the errors.

**Pull request merged**
